os/sys/cc.h

```diff
@@ -109,6 +109,10 @@
 #define ABS(n)      (((n) < 0) ? -(n) : (n))
 #endif
 
+#ifndef BOUND
+#define BOUND(a, minimum, maximum)   MIN(MAX(a, minimum), maximum)
+#endif
```

Comment:
Such macros may cause unintended side effects (think of `i++`). A safer option would be an inline function.

Reply:
We already have MIN, MAX, ABS macros in the same header that have the same behavior. Compared to inline functions, macros have the benefit that they can be used for compile-time computations, including assertions. And I'd hope that contributors to Contiki and reviewers are proficient enough in C programming to be aware of this and catch unintentional bugs.

If we decide to do switch to inline functions, best to do it in another PR that handles all of them at once.

Reply:
I agree with @atiselsts here